tedana/resources/decision_trees/minimal.json

```diff
@@ -194,11 +214,12 @@
             "kwargs": {
                 "var_metric": "variance explained",
                 "log_extra_info": "",
-                "log_extra_report": "",
+                "log_extra_report": "Any provisionally rejected components with variance explained less than ? are accepted.",
```

Comment:
@handwerkerd do you know how the parameters here can be described?

Reply:
I think this function gives an automatic description when run. The log_extra_report field is only useful if you want to give additional explanation.

The current automatic message will be something like
`accepted if variance explained<0.1. All variance<1.0, else nochange`

Reply:
Thanks for pointing me to the current message. That said, the `log_extra_report` field does show up in the HTML report, which is what I was hoping for. See https://output.circle-artifacts.com/output/job/500e9760-3014-49f4-a17e-a503ce9a1c32/artifacts/0/tmp/src/tedana/.testing_data_cache/outputs/five-echo/tedana_report.html for an example of the updated report text from this PR.